h5c/vic/src/vic-webapp/e2e/app.po.ts

```diff
@@ -119,42 +122,47 @@ export class VicWebappPage {
 
   openVchWizard() {
     this.switchFrame(this.iframeTabs);
-    this.waitForElementToBePresent(this.buttonNewVch);
+    this.waitForElementToBePresent(this.buttonNewVch, 'css');
     this.clickByCSS(this.buttonNewVch);
     browser.switchTo().defaultContent();
     this.switchFrame(this.iframeModal);
   }
 
   selectComputeResource(name: string = 'Cluster') {
     browser.sleep(defaultTimeout);
-    this.waitForElementToBePresent(this.datacenterTreenodeCaret);
+    this.waitForElementToBePresent(this.datacenterTreenodeCaret, 'css');
     element(by.xpath(this.firstDcTreenodeCaretXpath)).isPresent().then(collapsed => {
```

Comment:
Just a general question, do you know if finding by xpath vs css gives any benefits?